test/getUsers.spec.js

```diff
@@ -0,0 +1,98 @@
+const supertest = require('supertest');
+const { expect } = require('chai');
+const app = require('../app');
+const request = supertest(app);
+const { createUsers } = require('./helpers');
+const { host, port } = require('../config').common.api;
+const { AUTHENTICATION_ERROR, NOT_FOUND_ERROR } = require('../app/errors');
+
+describe('GET /users', () => {
+  it('should success when user is logged in', () =>
+    createUsers(25)
+      .then(() =>
+        request.post('/users/sessions').send({ email: 'mail1@wolox.com.ar', password: '1234587ocho' })
```

Comment:
Avoid use múltiple endpoints in same test, because this couples endpoints and is hard to modify, instead create a factory for tokens or something like that